### ISSUE TITLE
"This Hangouts extension is out of date. There's a newer version available."

### ISSUE THREAD

**Issue by ajayyy** [old-hangouts/old-hangouts-chrome-extension#2]
I decided to try downloading the old chromium and installing this extention there to make the extention have docking.i uninstalled the extention dn installed it on the old chromium, and now it says this message and does not let you use the extention. I tried reinstalling the extention on my main chrome and it does the same thing. It appears there is some timer built in that if you install after a certain time, it stop working. Now I'm stuck with no extention.

Tonorrow I'll try decompiling an older version of the hangouts extention and seeing if that works.

**Comment by femee24** [comment]
Same. I no longer have the old hangouts. Again :(

**Comment by ajayyy** [comment]
Why is google making it so hard to use this extention. It's crazy!

**Comment by ghost** [comment]
Same. It's taking up the whole extension window, making it unusable. :(

**Comment by ajayyy** [comment]
RunAsDate did not help. I will try the old extension now.

**Comment by fancellu** [comment]
Ditto. Same for me.

**Comment by FoxxMD** [comment]
Same. This is a sad day. The new app is terrible.

**Comment by ajayyy** [comment]
Why did I uninstall it... Now I can't get it again...

**Comment by ajayyy** [comment]
The old extention doesn't work, must be something with the servers.

**Comment by phavel** [comment]
Same.  :(

**Comment by scytheria1001** [comment]
Same. This is awful.

**Comment by fancellu** [comment]
Google: Don't be evil

http://time.com/4060575/alphabet-google-dont-be-evil/

Yep.

**Comment by ajayyy** [comment]
So I checked and it looks like the **actual** website where the old extension was hosted now just has that message. Looking deeper to see what I can do.

**Comment by ghost** [comment]
I wonder if there is any way to get the docked chat windows that you get in Inbox? Not sure how that would work.

**Comment by ajayyy** [comment]
@udpPigeons Right now I am trying to get the new hangout popup window html into the docked window.

**Comment by ajayyy** [comment]
@udpPigeons @old-hangouts I am trying to use the new interface's data in the old but am getting the error `TypeError: a.__onExtWindowLoad is not a function` even after importing every asset

**Comment by ghost** [comment]
Maybe importing the new interface data into the old creates conflicting assets? Don't know if that makes sense, or if that's possible.

**Comment by ajayyy** [comment]
@udpPigeons Looks like it's actually a new method with the update. I will try to consolidate the two

**Comment by ajayyy** [comment]
I'm giving up for now, might try more later. For now I'll use the popup window + an alwaysontop program.

**Comment by ghost** [comment]
I know it's not the same but you could use the Chrome Hangouts **app** (versus the extension) and use the option "Enable Transparent UI." That way it's on top and you get some floating chat bubbles. Not the same, I know, but better than the single window UI, IMO.

**Comment by sandoche** [comment]
@udpPigeons can you share the link of the Chrome Hangouts app ?

I will update the tutorial all the alternative solutions we found in the meanwhile.
It's very annoying that they closed the old extension.

**Comment by ajayyy** [comment]
@sandoche @udpPigeons  It's actually amazing!! https://chrome.google.com/webstore/detail/google-hangouts/knipolnnllmklapflnccelgolnpehhpl/related

**Comment by sandoche** [comment]
@ajayyy @udpPigeons I have it but (it's the extension) I don't find any "Enable Transparent UI" option or floating chat bubbles ?

The only alternative I found now is to use this extension and an always on top program I found two in Windows but the one in Mac seems not to work with my Mac OSX Sierra, I will investigate more later !

I updated the readme with the solutions for windows.

**Comment by ghost** [comment]
@sandoche The extension does not have the transparency option. Only the app does. They both look identical, but the options are different.

Here's a link to it: https://chrome.google.com/webstore/detail/google-hangouts/knipolnnllmklapflnccelgolnpehhpl

**Comment by ghost** [comment]
I wanted to add a separate post for getting the transparent UI.
1. Install the Chrome **app** (_not_ the extension) [following this link](https://chrome.google.com/webstore/detail/google-hangouts/knipolnnllmklapflnccelgolnpehhpl).
2. Click on the 3 horizontal bars in the top left hand corner (next to your name).
3. In the settings menu, scroll down to the "Hangouts App Settings" section.
4. Click the button "Enable transparent UI."
5. ?????
6. Profit.

**Comment by sandoche** [comment]
@udpPigeons actually I don't have this option but just found out that there is an Always on top option

<img width="620" alt="screen shot 2016-11-04 at 13 50 04" src="https://cloud.githubusercontent.com/assets/3635348/20005994/b20ad89a-a295-11e6-9988-e32ce3a98b97.png">

**Comment by ghost** [comment]
@sandoche It's not below the "Zoom" option?

![temp](https://cloud.githubusercontent.com/assets/22171142/20006362/9200ef58-a26d-11e6-8fc3-094a6b804f5e.png)

**Comment by mcbrineellis** [comment]
@udpPigeons - looks like @sandoche is using Mac OS (based on the lack of that option & the font shading in his screenshot).  "Transparent UI"/chat head option is a Windows-only thing, and has been since that option first came out with the Chrome App.

**Comment by ghost** [comment]
@mcbrineellis I suspected that, but didn't want to assume. Thank you for the clarification.

**Comment by ajayyy** [comment]
And, soon the chrome app will be killed...

**Comment by ajayyy** [comment]
Is this repo sadly dead now?

**Comment by pranav-prakash** [comment]
@ajayy Yup — the panels interface was removed form chrome entirely, and the api endpoints that the extension used to hit are removed.

**Comment by ajayyy** [comment]
@udpPigeons THEY DID IT! The Transparent UI feature on Windows is now gone :(

Edit: It's back
